spec/lib/bamboozled/api/employee_spec.rb

```diff
@@ -97,14 +109,13 @@
   end
 
   describe "#add" do
-    it 'creates a new employee in BambooHR' do
-
-      xml = YAML.load_file('spec/fixtures/add_employee_xml.yml')
-      response = File.new('spec/fixtures/add_employee_response.json')
-      details = JSON.parse(File.read('spec/fixtures/add_employee_details.json'))
+    it "creates a new employee in BambooHR" do
+      xml = YAML.load_file("spec/fixtures/add_employee_xml.yml")
+      response = File.new("spec/fixtures/add_employee_response.json")
+      details = JSON.parse(File.read("spec/fixtures/add_employee_details.json"))
 
-      stub_request(:post, /.*api\.bamboohr\.com.*/).
-        with(xml).to_return(response)
+      stub_request(:post, /.*api\.bamboohr\.com.*/)
+        .with(xml).to_return(response)
```

Comment:
This is a default Hound rule that will be overwritten with our custom `rubocop` rules.